solr/core/src/test/org/apache/solr/cloud/UnloadDistributedZkTest.java

```diff
@@ -65,7 +67,11 @@ protected String getSolrXml() {
 
   @Test
   public void test() throws Exception {
-    jettys.forEach(j -> j.getCoreContainer().getAllowPaths().add(Path.of("_ALL_"))); // Allow non-standard core instance path
+    jettys.forEach(j -> {
+      Set<Path> allowPath = j.getCoreContainer().getAllowPaths();
+      allowPath.clear();
+      allowPath.addAll(SolrPaths.ALL_PATHS); // Allow non-standard core instance path
```

Comment:
Do you think this Set should be mutable?  Hmm.  Not a blocker to me but worth contemplating.

Reply:
I was also surprised to see that. There are many tests (and only tests) that use this mutablity to add test dirs in the allowed list. I don't like that very much, but I don't think this should be changed in this PR.